eoscostarica/eos-rate
# Block Producer

### GH Block Producer 

#277
#180
#262 

- Refactor Slider rate component
- Update block-producer.js (use hook)
- Add tooltips property in react-chartjs-2 options
- Upgrade material UI package (Migration from v3 to v4)
- Upgrade react-chartjs-2 package
- Update material UI utils functions
- Validate block producer Profile
- Validate block producer list page
- Validate mobile version

#

### Steps to test

1. go to EOS Rate web app
2. login
3. go to block producers page (in this page to move mouse hover radar card and see tooltip)
4. choose one block producer and click it
5. click Rate this BP button (in this page to move mouse hover radar card and see tooltip)
6. see the new design for slider rate component

### Checklist
